### DISCUSSION TITLE
Error in installing photoshop 2021

### DISCUSSION THREAD

**Discussion by mochamellow** [LinSoftWin/Photoshop-CC2022-Linux#37] [Nothing to do here]
Got an error "touch: cannot touch '/progress.mimifile': Permission denied linux" when running "photoshop2021install.sh"

**Comment by MiMillieuh** [comment]
> Got an error "touch: cannot touch '/progress.mimifile': Permission denied linux" when running "photoshop2021install.sh"

Hello,

First please use issue next time. Discussion isn't made for support and all that stuff.

You can't install Photoshop where you don't have the permission. You need to give a folder that you have permission to.
